easybuild/easyconfigs/c/Clp/Clp-1.17.7-foss-2021b.eb

```diff
@@ -0,0 +1,61 @@
+easyblock = 'ConfigureMake'
+
+name = 'Clp'
+version = '1.17.7'
+
+homepage = "https://github.com/coin-or/Clp"
+description = """Clp (Coin-or linear programming) is an open-source linear programming solver.
+It is primarily meant to be used as a callable library, but a basic,
+stand-alone executable version is also available."""
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+source_urls = ['https://github.com/coin-or/Clp/archive/refs/tags/releases/']
+sources = ['%(version)s.tar.gz']
+checksums = ['c4c2c0e014220ce8b6294f3be0f3a595a37bef58a14bf9bac406016e9e73b0f5']
+
+builddependencies = [
+    ('Autotools', '20210726'),
+    ('Doxygen', '1.9.1'),
+    ('pkgconf', '1.8.0'),
+]
+
+dependencies = [
+    ('METIS', '5.1.0'),
+    ('MUMPS', '5.4.1', '-metis'),
+    ('CoinUtils', '2.11.6'),
+    ('Osi', '0.108.7'),
+    ('bzip2', '1.0.8'),
+    ('zlib', '1.2.11'),
+]
+
+# Use BLAS/LAPACK from OpenBLAS
+configopts = '--with-blas="$LIBBLAS" --with-lapack="$LIBLAPACK" '
+
+# Use METIS AND MUMPS from EB
+# --with-metis-lib is ignored
```

Comment:
is this correct? We have `--with-metis-lib` in `configopts`, and it does seem to be picked up